bindata/external-secrets/networkpolicy-allow-bitwarden-server-traffic.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
nit: can we rename the file to networkpolicy_allow-api-server-egress-for-bitwarden-sever, maintain the unfiromity.
it should <resourcekind_resourcename>
The same for all manifests.